api/server/routes/files/index.js

```diff
@@ -20,8 +19,7 @@ const initialize = async () => {
   router.post('/', upload.single('file'));
   router.post('/images', upload.single('file'));
 
-  router.use('/stt', stt);
-  router.use('/tts', tts);
+  router.use('/speech', speech);
```

Comment:
update your branch and you'll see an important note on route initialization order